src/app/auth/components/verify-email/verify-email.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import {ActivatedRoute, Router} from '@angular/router';
+import {Subscription} from 'rxjs';
+import {ApiConstants} from '../../../constants/api.constants';
+import {HttpClient} from '@angular/common/http';
+import {environment} from '../../../../environments/environment';
```

Comment:
imports сначала все внешние, делаем строчку пропуск и импортируем внутренние + отступы
импорты с абсолютным путем к файлу
```suggestion

import { Component, OnInit } from '@angular/core';
import { ActivatedRoute, Router } from '@angular/router';
import { HttpClient } from '@angular/common/http';
import { Subscription } from 'rxjs';

import { ApiConstants } from 'app/constants/api.constants';
import { environment } from 'environments/environment';
```